src/test/functional/tests/addUser.spec.ts

```diff
@@ -0,0 +1,27 @@
+import { expect, test } from '@playwright/test';
+
+import { adminUserTestData } from '../data';
+import { LoginPage } from '../pages/LoginPage';
+import { PimPage } from '../pages/PimPage';
+
+let loginPage: LoginPage, pimPage: PimPage;
+
+test.describe('Adding a new employee', () => {
+  test.beforeEach(async ({ page }) => {
+    loginPage = new LoginPage(page);
+    pimPage = new PimPage(page);
+    await loginPage.initialize();
+    await loginPage.loginUser(adminUserTestData.userName, adminUserTestData.password);
+    await loginPage.navigateToSubPage('PIM');
+    await expect(page.getByRole('heading', { name: 'Employee Information' })).toBeVisible();
```

Comment:
You could make an object with SubPage (enum), Header pairs and navigateToSubPage could:
`await page.getByRole('heading', { name: header }).waitFor({ state: "visible" });`
after opening a page - and as a rule I don't think we should use expects (assertions) in places other than tests or methods specifically designed to do assertions (reusable data/forms comparators etc). Locator.waitFor will do the job just fine without beeing "test assertion". As a rule of thumb - assertions should be last step of tests (ergo there should be no other actions after assertions).